README.md

```diff
@@ -87,6 +87,14 @@ models. The
 [server](https://github.com/triton-inference-server/server/tree/master/src/backends/pytorch)
 repo contains the source for the backend.
 
+**Python**: The Python backend allows you to write your model logic in
+Python. For example, you can use this backend to execute pre/post
+processing code written in Python, or to run a PyTorch script directly
```

Comment:
'or use PyTorch library instead of using TorchScript'. I guess maybe because PyTorch already has 'Py' in it, we can probably rename it to libtorch or simply 'torch' backend to avoid confusion.

Reply:
@CoderHam  we should consider using torchscript_backend as the repo name when we move the backend.